src/UnitTest/test_quote_prov.cpp

```diff
@@ -1061,11 +1037,11 @@ TEST(testQuoteProv, quoteProviderTestsV3DataFromService)
     // Get the data from the service
     //
     SetupEnvironment("v3");
-    ASSERT_TRUE(RunQuoteProviderTests());
-    ASSERT_TRUE(RunQuoteProviderTestsICXV3());
-    ASSERT_TRUE(GetQveIdentityTest());
+   // ASSERT_TRUE(RunQuoteProviderTests());
```

Comment:
Probably want to uncomment these tests so we have v3 tests running for qve etc.

Reply:
Added ICVX3 test. Had to change QEID to non sbx part.